examples/flex-flows/unify-ai/code_quality_unify_ai.py

```diff
@@ -0,0 +1,19 @@
+import pathlib
+import sys
+
+from promptflow.core import OpenAIModelConfiguration
+
+# Add the path to the evaluation code quality module.
+eval_code_quality_path = str(pathlib.Path(__file__).parent / "../eval-code-quality")
+sys.path.append(eval_code_quality_path)
+
+import code_quality as code_quality_aoi
+
+
+class CodeEvaluator(code_quality_aoi.CodeEvaluator):
```

Comment:
Semms this eval flow does the same thing with `eval-code-quality`? I suggest directly use  `eval-code-quality` instead of creating a new one

Reply:
Had to create a new one as model_config type has changed. I need to use `OpenAI` client instead of `AzureOpenAI` python client. Although python shouldn't care about type hints, promptflow in execution flow seems to be using type hints to determine type of config. So to use OpenAI client in the flow we have to define mode_config type hint `OpenAIModelConfiguration` instead of `AzureOpenAIModelConfiguration`